lib/find.coffee

```diff
@@ -72,6 +72,7 @@ module.exports =
         @projectFindPanel.hide()
       else
         @projectFindPanel.show()
+        @projectFindView.focusFindElement()
```

Comment:
Rather than just adding the one missing line, maybe we could turn these two implementations into a common function so that find and find-in-project always work the same?